hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -32,10 +32,14 @@
 
   private static final double DEFAULT_JVM_OVERHEAD = 1.35;
   private static final int DEFAULT_JOURNAL_NODE_COUNT = 3;
+  private static final int DEFAULT_DATA_NODE_COUNT = 3;
   private static final int DEFAULT_FAILOVER_TIMEOUT = 31449600;
   private static final int DEFAULT_ZK_TIME_MS = 20000;
   private static final int DEFAULT_RECONCILIATION_TIMEOUT = 30;
   private static final int DEFAULT_DEADNODE_TIMEOUT = 90;
+  private static final int DEFAULT_DFS_REPLICATION = 3;
+  @SuppressWarnings("PMD")
+  private static final String DEFAULT_SERVER_HOST = "0.0.0.0";
 
```

Comment:
If we are going to create one string as a constant here, we need to make all our strings constants. Perhaps we could create another constants file to separate out this logic. If not, we need to be consistent, and we shouldn't create constants for String values.